themes/custom/az_barrio/az_barrio.info.yml

```diff
@@ -3,8 +3,8 @@ type: theme
 description: 'Basic structure for a Bootstrap Barrio SubTheme.'
 core_version_requirement: ^8.8 || ^9
 base theme: bootstrap_barrio
-logo: az_barrio/logo.png
-footer_logo: az_barrio/footer_logo.png
+logo: az_barrio/images/logo.png
```

Comment:
Looks like we don't need the theme name here according to this https://www.drupal.org/docs/8/theming-drupal-8/defining-a-theme-with-an-infoyml-file#s-introduced-in-86x